src/components/ProfilePage/AddressBox/AddressBox.style.ts

```diff
@@ -18,12 +18,15 @@ export const AddressBoxContainer = styled(Box, {
   position: 'relative',
   paddingBottom: theme.spacing(1),
   width: '100%',
+  minHeight: '200px',
```

Comment:
No importan, but generally: Let´s use numbers in a styled component and not add "px".

But be aware, using numbers in an sx-prop will actually use spacings (n * 8px). There we must add "px".